playbook/roles/varnish/templates/default.vcl.j2

```diff
@@ -268,6 +268,18 @@ sub vcl_recv {
     return (pass);
   }
 
+
+  {% if varnish.recv_conditions is defined %}
+
+  # Conditions specified by variables:
+  {% for condition in varnish.recv_conditions %}
+  if (req.{{ condition.property }} {{ condition.operator }} "{{ condition.value }}") {
+    return ({{ condition.return_value }});
```

Comment:
We could also leave the full definition of action to the variables instead of only supporting simple "return"

Reply:
alright!